contracts/axelar-gas-service/tests/test.rs

```diff
@@ -221,56 +210,57 @@ fn add_gas() {
 }
 
 #[test]
-fn fail_collect_fees_zero_refund_amount() {
+fn collect_fees_fails_with_zero_refund_amount() {
```

Comment:
```suggestion
fn collect_fees_fails_with_zero_amount() {
```since refund is another method